modules/windows-pod-placement.adoc

```diff
@@ -7,6 +7,12 @@
 
 Before deploying your Windows workloads to the cluster, you must configure your Windows node scheduling so pods are assigned correctly. Since you have a machine hosting your Windows node, it is managed the same as a Linux-based node. Likewise, scheduling a Windows pod to the appropriate Windows node is completed similarly, using mechanisms like taints, tolerations, and node selectors.
 
-With multiple operating systems, and the ability to run multiple Windows OS variants, in the same cluster, you must map your Windows pods to a base Windows OS variant by using a `RuntimeClass`. For example, if you have multiple Windows nodes running on different Windows Server container versions, the cluster could schedule your Windows pods to an incompatible Windows OS variant. You must have `RuntimeClass` objects configured for each Windows OS variant on your cluster. Using a `RuntimeClass` object is also recommended if you have only one Windows OS variant available in your cluster.
+With multiple operating systems, and the ability to run multiple Windows OS variants in the same cluster, you must map your Windows pods to a base Windows OS variant by using a `RuntimeClass`. For example, if you have multiple Windows nodes running on different Windows Server container versions, the cluster could schedule your Windows pods to an incompatible Windows OS variant. You must have `RuntimeClass` objects configured for each Windows OS variant on your cluster. Using a `RuntimeClass` object is also recommended if you have only one Windows OS variant available in your cluster.
 
 For more information, see Microsoft's documentation on link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/update-containers#host-and-container-version-compatibility[Host and container version compatibility].
+
+[IMPORTANT}
+====
+The container base image must be the same Windows OS variant and build version that is running on the node.
```

Comment:
In the original BZ there is a suggestion to point to the official Microsoft docs for more information: 

> Can I request that this be treated as a doc BZ and update our doc (below) to clarify that the container base image must match the windows version running on the OCP node and that the build number must match and as an additional data refer to Microsoft docs.   

  


 https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility
 , maybe you could add a reference to it in case more information is expected.